pkgs/development/python-modules/isosurfaces/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+}:
+
+buildPythonPackage rec {
+  pname = "isosurfaces";
+  version = "0.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "fa1b44e5e59d2f429add49289ab89e36f8dcda49b7badd99e0beea273be331f4";
+  };
+
+  buildInputs = [
+    numpy
+  ];
+
+  meta = with lib; {
+    homepage = "https://github.com/jared-hughes/isosurfaces";
+    description = "Construct isolines/isosurfaces of a 2D/3D scalar field defined by a function. ";
```

`meta.description` should not end with a period.
```suggestion
    description = "Construct isolines/isosurfaces of a 2D/3D scalar field defined by a function";
```

---

done!